docs/reference/setup/install.asciidoc

```diff
@@ -3,10 +3,11 @@
 
 [float]
 === Hosted Elasticsearch
-Elasticsearch can be run on your own hardware or using our hosted
-Elasticsearch Service on https://www.elastic.co/cloud[Elastic Cloud], which is
-available on AWS and GCP. You can
-https://www.elastic.co/cloud/elasticsearch-service/signup[try out the hosted service] for free.
+Elasticsearch can be run on your own hardware or using our hosted Elasticsearch
+Service on https://www.elastic.co/cloud[Elastic Cloud], which is available on
+AWS, GCP and Azure. You can
```

Comment:
Unrelated except that I was looking for other references to Cloud: reformat & mention Azure.